src/mpievent.jl

```diff
@@ -29,3 +30,13 @@ function getsrcdest(ev::MPIEvent)
     end
     return nothing
 end
+
+function gettag(ev::MPIEvent)
+    if isempty(ev.args_subset)
+        return nothing
+    end
+    if haskey(ev.args_subset, :tag)
```

Comment:
Verstehe die tag Geschichte nicht ganz. Ist `haskey(ev.args_subset, :tag)` aktuell nicht einfach immer `false`?

Reply:
Ich nutze die Funktion beim Matchen der MPI Events. Im Moment hat sie keinen Effekt, da sie immer nothing zurück gibt, aber sobald wir tags in unserem NamedTuple hinzufügen, sollten sie damit auch direkt berücksichtigt werden.